src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -104,7 +104,7 @@ public WebView() : this(Interop.WebView.New(), true)
         /// <param name="locale">The locale language of Web</param>
```

Comment:
in the previous line, did you mean "locale language", I think you can drop the change: "local language" -> "specified language" / "specified language locale"

Reply:
Done.